src/mercury_prof_interface.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2013-2020 Argonne National Laboratory, Department of Energy,
+ *                    UChicago Argonne, LLC and The HDF Group.
+ * All rights reserved.
+ *
+ * The full copyright notice, including terms governing use, modification,
+ * and redistribution, is contained in the COPYING file that can be
+ * found at the root of the source code distribution tree.
+ */
+
+#ifndef MERCURY_PROF_INTERFACE_H
+#define MERCURY_PROF_INTERFACE_H
+
+#include "mercury_prof_types.h"
+
+/*Initialize and finalize routines */
+HG_PUBLIC hg_return_t HG_Prof_init();
```

Comment:
Could you please add documentation for every function using the template that is similar to other mercury headers? i.e. something like:
```C
/**
 * <Description>
 *
 * \param param1 [IN/OUT]         <description>
 * \param param2 etc
 *
 * \return HG_SUCCESS or corresponding HG error code
 */
```
Since you added comments in the .c instead, you could just move those comments to the header. We usually don't duplicate function documentation between source and header so having it in one place in the header would be best.

Reply:
@soumagne: I have addressed this issue in the latest commit.

Reply:
Great thanks! Yes that's much easier to review the API.